app/models/stop.rb

```diff
@@ -284,6 +291,52 @@ def clean_attributes
   end
 end
 
+class StopStation < Stop
+  current_tracked_by_changeset({
+    kind_of_model_tracked: :onestop_entity,
+    virtual_attributes: [
+      :served_by,
+      :not_served_by,
+      :identified_by,
+      :not_identified_by,
+      :imported_from_feed
+    ]
+  })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.